instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integrationtests/profiling/MemoryProfileForLogs.kt

```diff
@@ -21,6 +22,7 @@ internal class MemoryProfileForLogs {
     val memoryProfilingRule = MemoryProfilingRule()
 
     @Test
+    @Ignore
```

Comment:
Can you add a // TODO RUMM-164, and explain the reason for those tests being ignored in @Ignore("…")